docs/tasks.py

```diff
@@ -11,7 +11,8 @@ def html(context, sourcedir="source", builddir="build"):
         sourcedir (str, optional): Source directory for sphinx to use. Defaults to "source".
         builddir (str, optional): Output directory for sphinx to use. Defaults to "build".
     """
-    command = "sphinx-build" + sourcedir + builddir
+    print("Building html documentation...")
+    command = "sphinx-build " + sourcedir + " " + builddir
```

Comment:
We seem to use f-strings in our tasks.py

Reply:
Updated